docs/4.1. Node to Authorization Server Interactions.md

```diff
@@ -227,4 +227,4 @@ Content-Type: application/json
 ]
 ```
 
-Note that if a Node is unable to contact an Authorization Server, it should attempt to contact another Authorization Server from the discovered list until this [either succeeds or the list is exhausted](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
+Note that if you have multiple Authorization Servers there are some extra things to consider about [management of their corresponding public keys](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
```

Comment:
I think the wording isn't quite right wrong here. 'You' is generally the implementer in this guide, but having many authorization servers is a deployment scenario. Either the spec says this is something all implementations are required /recommended to handle, or not?

Reply:
LGTM